sdk/kusto/Azure.ResourceManager.Kusto/api/Azure.ResourceManager.Kusto.netstandard2.0.cs

```diff
@@ -547,6 +549,19 @@ public partial class AcceptedAudience
         public static bool operator !=(Azure.ResourceManager.Kusto.Models.BlobStorageEventType left, Azure.ResourceManager.Kusto.Models.BlobStorageEventType right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial class CosmosDbDataConnection : Azure.ResourceManager.Kusto.KustoDataConnectionData
+    {
+        public CosmosDbDataConnection() { }
+        public string CosmosDbAccountResourceId { get { throw null; } set { } }
```

Comment:
Could you please double confirm the type of this property? Is it an arm id, which will be formatted like `/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{resourceProviderNamespace}/{resourceType}/{resourceName}`? We need specific types in .NET SDK and [ResourceIdentifier](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/ResourceIdentifier.cs) will be used here if the property is an arm id.

Reply:
CosmosDbAccountResourceId  is /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{resourceProviderNamespace}/{resourceType}/{resourceName} format.

how do you want me to declare it? 
is it auto generated from swagger?
How do prevent override upon next generation ?